### ISSUE TITLE
SourceMaps for Less workflow

### ISSUE THREAD

**Issue by gbmoretti** [half-pipe/half-pipe#64]
As discussed previously in #61 I made a quick research and apparently [grunt-csso](https://github.com/t32k/grunt-csso) doesn't support sourceMaps.
Another good package we could use to minify stylesheets is [grunt-contrib-cssmin](https://github.com/gruntjs/grunt-contrib-cssmin) but it doesn't support sourceMaps yet too gruntjs/grunt-contrib-cssmin#58

**Comment by MarkMurphy** [comment]
I like grunt-contrib-cssmin and we should switch to that. I don't think it's necessary to minify the css during development anyway it adds extra overhead so I think it should be removed from the debug task.

We can and an additional task that does include minification but warns of this issue. Once support for source maps is added to the minifier we can remove the warning and still leave them with the option to minify or not with the default being un-minified. 

How's that sound?

**Comment by gbmoretti** [comment]
You are right, minification only makes sense in a production environment and sourceMaps in development.

I'll remove the minify task from debug by now.

**Comment by gbmoretti** [comment]
I guess I'll can do it only by monday. If anyone could do it before, I don't mind.

**Comment by octavioamu** [comment]
Maybe a solution (for now) for source maps and others previous tasks without this option, could be https://npmjs.org/package/grunt-concat-sourcemap using this like a source map generator. Also I didn't test, and don't know if it works with css files.
Looking at the code, looks like it works with css files too https://github.com/kozy4324/grunt-concat-sourcemap/blob/master/tasks/concat_sourcemap.js
